hash/c64_cass.xml

```diff
@@ -16333,6 +16357,7 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
 		<description>Yie Ar Kung-Fu</description>
 		<year>1985</year>
 		<publisher>Erbe</publisher>
+		<info name="usage" value="Ensure c1541 Slot Device is removed prior to loading"/>
```

Comment:
Can you use something like `<sharedfeat name="iec8_default" value=""/>` to make it automatically empty the slot, or is that problematic with this software list?

Reply:
I have spent quite some time trying to get this to work and I can't seem to change the iec8 from the default "c1541".  I understand this approach works well with other devices like the a2600 but doesn't work with the C64.